include/aie/detail/fiber/pooled_shared_work.hpp

```diff
@@ -0,0 +1,177 @@
+//          Copyright Nat Goodspeed + Oliver Kowalke 2015
```

Comment:
It looks like you copy-paste a lot of stuff from `triSYCL/detail` which increase technical debt.
Just use the files from there.

Reply:
triSYCL is not ACAP++ and a lot of stuff is in `triSYCL/detail`.
Furthermore ACAP++ is not removed yet, so one little step after another.